Fix cursor size

Sporif opened the pull request:
Default to a size of 24 pixels and use the environment variable XCURSOR_SIZE if it's set.

Closes #20

lbonn commented on source/wayland/display.c:
```diff
@@ -1208,8 +1208,18 @@ static gboolean wayland_display_setup(GMainLoop *main_loop,
+  if (env_cursor_size && strlen(env_cursor_size) > 0) {
+      errno = 0;
+      char *end;
+      unsigned size = strtoul(env_cursor_size, &end, 10);
```
There is the glib function `g_ascii_strtoull` that can be used here https://www.freedesktop.org/software/gstreamer-sdk/data/docs/2012.5/glib/glib-String-Utility-Functions.html.

Then we can just check that the return is different from `G_MAXUINT64` and 0 (cursor size of 0 is unlikely...) and avoid messing with errno and `char *end`

Sporif replied:
How about now?

lbonn replied:
Nice, thank you

Sporif committed:
Fix cursor size

Default to a size of 24 pixels
Use the environment variable XCURSOR_SIZE if it's set
